src/domain/request/usecase/ExecuteReadyRequestsUseCase.spec.ts

```diff
@@ -52,55 +49,58 @@ describe('ExecuteReadyRequestsUseCase', () => {
 
   it('should execute ready request which is finished after that and generate response', async () => {
     // given
+    const responseRepository = new InMemoryResponseRepositoryAdapter();
     const sut = new ExecuteReadyRequestsUseCase(
-      fetchDataUseCase(),
-      selectDataUseCase(),
-      sendResponseToOracleUseCase(),
-      requestRepository(),
-      responseRepository(),
-      new Logger(),
+      fetchDataUseCase() as unknown as FetchDataUseCase,
```

Comment:
Maybe we could declare these types in the variable declaration? This way we wouldn't have to repeat this code when creating `sut` each time.

Reply:
Or other solution. Instead of `x as unknown as X` to `x as Partial<X>` generic partial is build in TS